modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java

```diff
@@ -88,6 +88,11 @@ private void checkSciNotation(DD x, String expectedStr) {
 	}
 
 	public void testParse() {
+		//checkParse("0", 0, 1e-32);
```

Comment:
Any reason this is commented?

Reply:
Problem calculating relerr (div/0), fixed that.